clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/AbstractStorageEngine.java

```diff
@@ -173,7 +151,7 @@ protected final synchronized void restoreStoragePartitions() {
   }
 
   // For testing purpose only.
-  public AbstractStoragePartition getMetadataPartition() {
+  protected AbstractStoragePartition getMetadataPartition() {
```

Comment:
no nit/issue just a question.  Do we think we'll do a similar refactor for AbstractStoragePartition?  I'm looking at the class now.  Honestly, I think it could probably just become an interface now, it's a pretty barebones set up.  It's only the partitionId that prevents it from being so.